dynamiqs/smesolve/sme_solver.py

```diff
@@ -15,24 +15,27 @@ def __init__(
         self,
         *args,
         jump_ops: Tensor,
-        meas_ops: Tensor,
         etas: Tensor,
         generator: torch.Generator,
         t_meas: Tensor,
     ):
-        self.meas_ops = meas_ops  # (len(meas_ops), n, n)
-        self.etas = etas  # (len(meas_ops))
-        self.generator = generator
         self.t_meas = t_meas  # (len(t_meas))
-
         super().__init__(*args, jump_ops=jump_ops)
 
+        # split jump operators between purely dissipative (eta = 0) and
+        # monitored (eta != 0)
+        mask = etas == 0.0
+        self.T = jump_ops[mask]  # (len(T), n, n) purely dissipative
+        self.V = jump_ops[~mask]  # (len(V), n, n) monitored
```

Comment:
Are these notations from somewhere? IMO they suggest kinetic (T) and potential (V) energy terms. Could we use something else? Maybe `self.J` (jump) or `self.C` (collapse) and `self.M` (measured/monitored)?

Reply:
`self.M` conflicts with `Rouchon` solver `self.M0/self.M1s` and with the measurement back-action superoperator `\mathcal{M}`. 😕 I picked `Lc` (for collapse) and `Lm` (for monitored), tell me if that works for you.

Reply:
Perfect !